src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -524,12 +524,15 @@ private void prepareLeaderElectionStatements(final String timeUdf) {
     takeLeadPrepStmt = session
         .prepare(
             "INSERT INTO leader(leader_id, reaper_instance_id, reaper_instance_host, last_heartbeat)"
-                + "VALUES(?, ?, ?, " + timeUdf + "(now())) IF NOT EXISTS USING TTL ?");
+                + "VALUES(?, ?, ?, " + timeUdf + "(now())) IF NOT EXISTS USING TTL ?")
+        .setConsistencyLevel(ConsistencyLevel.QUORUM);
```

Comment:
is any of the leader election usages cross-dc (and need to be QUORUM over LOCAL_QUORUM) ?

Reply:
Yes, sadly we need cross DC consistency on leader election otherwise two Reaper instances in different DCs might be able to take the lead on the same segment.